src/pages/Events/EventDoorCounter.tsx

```
@@ -0,0 +1,210 @@
+import React, { useState, useEffect } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)